### PR TITLE
bacon : new receipe

### PR DESCRIPTION
**Pull request by yann64**
Receipe for BaCon : the Basic to C Converter.

### CODE REVIEW

File: dev-lang/bacon/bacon-4.7.2.recipe [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+different platforms (both 32bit and 64bit), while trying to revive \
+the days of the good old BASIC."
+HOMEPAGE="https://chiselapp.com/user/bacon/repository/bacon/home"
+COPYRIGHT="2009-2023 Peter van Eerten"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://chiselapp.com/user/bacon/repository/bacon/attachdownload/bacon-$portVersion.tar.gz?page=Downloads&file=bacon-$portVersion.tar.gz"
+CHECKSUM_SHA256="619b721c820e1b11314cd7b88da39db3e76d9063605ee98aa35fb3fe635ab06c"
+
+ARCHITECTURES="all"
```
Comment (Begasus) [inline dev-lang/bacon/bacon-4.7.2.recipe:15]: This probably wont work on 32bit, libfltk is secondary arch only there, so needs secondaryArchSuffix and the likes here :)

Comment (yann64) [reply dev-lang/bacon/bacon-4.7.2.recipe:15]: Ok, will check over the weekend as I am not yet familiar with that (and will amend the receipe accordingly).

Also got your messages on IRC:

- New version got published today, that is why I missed it. I was just unlucky with the release timing.
- Good point for the menu entry. Will also check on that over the weekend.

**Comment by yann64** [comment]
Went the easy way and ignored the GUI (for now) as I only use cli anyway.

File: dev-lang/bacon/bacon-4.8.recipe [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	cmd:make
+	cmd:awk
+	cmd:flex
+	cmd:indent
+	cmd:gcc
+	"
+
+BUILD()
+{
+	runConfigure ./configure --prefix=$binDir --datadir=$dataDir
```
Comment (Begasus) [inline dev-lang/bacon/bacon-4.8.recipe:39]: prefix and datadir shouldn't be neededd if you use runConfigure

**Comment by Begasus** [comment]
Build failure for both arch's, source uri changed?

**Comment by jmairboeck** [comment]
The error message suggests that their server certificate is expired or something.

**Pull request merged**